src/flags.ts

```diff
@@ -170,92 +151,105 @@ export class LDFlagManager implements IFlagManager {
 		};
 	}
 
-	registerProviders(ctx: vscode.ExtensionContext, settings: IConfiguration) {
+	registerProviders(ctx: ExtensionContext) {
```

Comment:
This method shouldn't be part of the `FlagStore`. I'm going to factor it out in another PR.